docs/api.md

```diff
@@ -408,6 +409,40 @@ It accepts a single input: `{ forwardSubscription }`. This is a function that
 receives an enriched operation and must return an Observable-like object that
 streams `GraphQLResult`s with `data` and `errors`.
 
+### `ssrExchange` (Exchange factory)
+
+The `ssrExchange` as [described in the Basics section](basics.md#server-side-rendering).
+It's of type `Options => Exchange`.
+
+It accepts a single input, `{ initialState }`, which is completely
+optional, which populates the server-side rendered data with
+a rehydrated cache.
+
+This can be used to extract data that has been queried on
+the server-side, which is also described in the Basics section,
+and is also used on the client-side to restore server-side
+rendered data.
+
+When called this function creates an `Exchange`, which also has
+two methods on it:
+
+- `.restoreData(data)` which can be used to inject data, typically
+  on the client-side.
+- `.extractData()` which is typically used on the server-side to
+  extract the server-side rendered data.
+
+This is basically a small cache that collects data on the server
```

Comment:
Replace paragraph with:

Basically, `ssrExchange` is a small cache that collects data during the server rendering pass, and allows you to populate the cache on the client-side with the same data.